python/flink_agents/integrations/chat_models/openai/utils.py

```diff
@@ -0,0 +1,131 @@
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+#################################################################################
+import json
+import os
+from typing import List, Optional, Sequence, Tuple, cast
+
+import openai
+from openai.types.chat import ChatCompletionMessage, ChatCompletionMessageParam
+
+from flink_agents.api.chat_message import ChatMessage, MessageRole
+
+DEFAULT_OPENAI_API_BASE_URL = "https://api.openai.com/v1"
+DEFAULT_OPENAI_API_VERSION = ""
+
+
+def resolve_openai_credentials(
```

Comment:
We have too many `utils.py` files. Just in this PR, there are two of them. Let's give them some meaningful names.